File: extensions/panache/hibernate-orm-panache-kotlin/deployment/src/test/java/io/quarkus/hibernate/orm/panache/kotlin/deployment/test/multiple_pu/first/FirstEntity.java
package io.quarkus.hibernate.orm.panache.kotlin.deployment.test.multiple_pu.first;

import javax.persistence.Entity;

import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity;

@Entity
public class FirstEntity extends PanacheEntity {

    public String name;
}

File: extensions/panache/hibernate-orm-panache-kotlin/deployment/src/test/kotlin/io/quarkus/hibernate/orm/panache/kotlin/deployment/test/multiple_pu/first/FirstEntity.kt
package io.quarkus.hibernate.orm.panache.kotlin.deployment.test.multiple_pu.first

import io.quarkus.hibernate.orm.panache.kotlin.PanacheEntity
import javax.persistence.Entity

@Entity
class FirstEntity : PanacheEntity() {
    var name: String? = null
}
